sycl/include/CL/sycl/INTEL/esimd/detail/emu/esimd_emu_functions_v1.h

```diff
@@ -11,6 +11,7 @@
 /// \ingroup sycl_pi_esimd_cpu
 
 #pragma once
+#include <cstdint>
```

Comment:
Including something here seems not very correct as contents of this file should be inserted as is to the structure on `esimdcpu_device_interface.hpp:48`. Suggest including the header in the `esimdcpu_device_interface.hpp` and have a comment here saying that this header is expected to be included by the includer.

Reply:
Will do.